contracts/strategies/LPOptimiserStrategy.sol

```diff
@@ -43,11 +57,133 @@ contract LPOptimiserStrategy is DefaultAccessControl, ILpCallback {
         _erc20Vault = erc20vault_;
         _vault = vault_;
         _marginEngine = IMarginEngine(vault_.marginEngine());
+        _periphery = IPeriphery(vault_.periphery());
+        _vamm = IVAMM(vault_.vamm());
         _currentPosition = vault_.currentPosition();
         _tokens = vault_.vaultTokens();
         _pullExistentials = vault_.pullExistentials();
     }
 
+    event Rebalanced(int24 newTickLowerMul, int24 newTickUpperMul);
+
+    function setConstants(int24 logProx, uint256 sigmaWad, uint256 max_possible_lower_bound, int24 tickSpacing) public {
+        _requireAtLeastOperator();
+        _logProximity = logProx;
+        _sigmaWad = sigmaWad;
+        _max_possible_lower_bound = max_possible_lower_bound;
+        _tickSpacing = tickSpacing;
+    }
+
+    function setTickValues(int24 currentTick, int24 tickLower, int24 tickUpper) public {
```

Comment:
I don't see where this function is going to be used.

Reply:
I wrote this function so I could set the required tick values in the tests. Effectively, once everything is wired up to the vault when merged this function would become obsolete.